mergify_engine/actions/dismiss_reviews.py

```diff
@@ -0,0 +1,55 @@
+# -*- encoding: utf-8 -*-
+#
+#  Copyright © 2018 Mehdi Abaakouk <sileht@sileht.net>
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import github
+
+import voluptuous
+
+from mergify_engine import actions
+
+
+class DismissReviewsAction(actions.Action):
+    validator = {
+        voluptuous.Required("approved", default=[]):
```

Comment:
I'd think we'd set `True` by default so if the action is enabled then it dismisses everything by default.